app/presenters/mainstream_browse_page_presenter.rb

```diff
@@ -9,7 +9,6 @@ def format
 
   def links
     super.merge(
-      "related_topics" => @tag.topics.order(:title).map(&:content_id),
```

Comment:
It looks like we missed removing `related_topics` from collections.  https://github.com/search?q=repo%3Aalphagov%2Fcollections%20related_topics&type=code

@georges1996 please could you open a PR to remove that code from collections first, and link it here? Reason being - the process we've followed so far is to remove any dependency on specialist topics from the frontend first, so that we can feel confident that when these fields are missing from the content items nothing will blow up. So we should stick with that process. 

@unoduetre please could you remove `related_topics` from [these examples](https://github.com/search?q=repo%3Aalphagov%2Fpublishing-api%20related_topics&type=code) in publishing API. There might be a dependency here so that George's PR in collections has to go live first.